v3/ashwing66/ashwing66.json

```diff
@@ -0,0 +1,404 @@
+{
+   "name":"Ashwing66",
+   "vendorId":"0x7372",
+   "productId":"0x6F64",
+   "menus":[
+      "qmk_rgb_matrix"
```

Comment:
![image](https://github.com/the-via/keyboards/assets/17491233/69b9585c-6000-498e-a7db-3300465b74e6)

I see you only have some of the animations enabled, I recommend you either change the VIA one to custom (and only enable the default ones), or add in ALL rgb effects in QMK.

Reply:
Made relevant changes to match whats allowed within QMK firmware.